.gitignore

```diff
@@ -51,4 +51,6 @@ ci/authn-k8s/dev/policies/*.yml
 # Vim backup files
 *.sw[po]
 
-
+#ignore audit table
```

Comment:
Why do we need this?

Reply:
We need to ignore the audit engine db migration files. that we will not mistakenly commit those link file

Reply:
In which flow we have those files in git?

Reply:
When starting conjur on development mode. we binned to code with our container, this cause the links file to created on our source code.

Reply:
Looks good.